src/pudl/analysis/record_linkage/classify_plants_ferc1.py

```diff
@@ -142,6 +150,15 @@ def merge_steam_fuel_dfs(
     ).astype({"plant_type": str, "construction_type": str})
 
 
+@graph
+def ferc_to_ferc(
+    input_df: pd.DataFrame, experiment_tracker: model_helpers.ExperimentTracker
+) -> pd.DataFrame:
+    """Graph implements core portion of the ferc-ferc plant matching model."""
+    feature_matrix = ferc_dataframe_embedder(input_df, experiment_tracker)
+    return link_ids_cross_year(input_df, feature_matrix)
+
+
 @graph_asset(config=model_helpers.get_model_config("ferc_to_ferc"))
```

Comment:
Maybe this is unrelated to the experiment tracking, but what does this Dagster config do? It gets a dictionary with the config for the ferc to ferc experiment and that's used by mlflow?

Reply:
Yeah I have a yaml file in `package_data/` that has configuration for the `ferc_to_ferc` model. Passing it to the `graph_asset` decorator will override the default values defined in `classify_plants_ferc1.py`. If you configure an `ExperimentTracker` with `log_yaml=True`, it will call `get_model_config` with the experiment name to look for configuration and log those parameters with mlflow